templates/php-component/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:7-cli
+
+ARG COMPOSER_FLAGS="--prefer-dist --no-interaction"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV COMPOSER_ALLOW_SUPERUSER 1
+
+WORKDIR /code/
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        git \
+        unzip \
+	&& rm -r /var/lib/apt/lists/* \
+	&& curl -sS --fail https://getcomposer.org/installer | php \
+	&& mv ./composer.phar /usr/local/bin/composer
```

Comment:
https://github.com/keboola/component-generator/blob/master/templates/php-keboola/docker/composer-install.sh ?

Reply:
Jo, pravda. Nelíbilo se mi, že se composer maže po tom, co se nainstalují závislosti (takže pak nejde `composer tests` třeba). Ale jde to i bez toho. Plus jsem to upravil tak, že místo wgetu je tam curl, takže to nepotřebuje instalovat wget ;)

Reply:
Jo, sakra! Já neměl pullnuto!

Reply:
V masteru už je to fixnuté!